persistence/remote/src/test/resources/remote-cl-uri-config.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-config-${infinispan.core.schema.version}.xsd
+                          urn:infinispan:config:store:remote:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-cachestore-remote-config-${infinispan.core.schema.version}.xsd"
+      xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+      xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}" >
+
+   <!-- Default cache named to preserve old default cache name -->
+   <cache-container default-cache="RemoteStoreConfigUriTest">
+      <local-cache name="RemoteStoreConfigUriTest">
+         <persistence passivation="false">
+            <remote-store
+	xmlns="urn:infinispan:config:store:remote:${infinispan.core.schema.version}"
+	cache="RemoteStoreConfigUriTest" fetch-state="false" preload="false"
+	purge="false" shared="true" read-only="false" hotrod-wrapping="false"
+	raw-values="false" socket-timeout="60000" tcp-no-delay="true"
+	balancing-strategy="org.infinispan.client.hotrod.impl.transport.tcp.RoundRobinBalancingStrategy"
+	transport-factory="org.infinispan.client.hotrod.impl.transport.netty.ChannelFactory"
+	key-size-estimate="62" value-size-estimate="512"
+	force-return-values="false" segmented="false"
+	uri="hotrods://127.0.0.1:19711?use_ssl=false&amp;tcp_no_delay=true&amp;tcp_keep_alive=false&amp;force_return_values=false">
+               <connection-pool max-active="99" max-pending-requests="97" min-idle="98"/>
+               <async-executor factory="org.infinispan.client.hotrod.impl.async.DefaultAsyncExecutorFactory">
+                  <property name="pool_size">1</property>
+                  <property name="queue_size">10000</property>
+               </async-executor>
+            </remote-store>
```

Comment:
As we're only really interested in testing the `uri` attribute, we can remove the config elements/attributes that aren't relevant to the `RemoteStoreConfigUriTest.java`.

I would just include the following for the remote-store definition and remove the child elements:

```
<remote-store uri="hotrods://127.0.0.1:19711?use_ssl=false&amp;tcp_no_delay=true&amp;tcp_keep_alive=false&amp;force_return_values=false" />
```